src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -302,6 +302,21 @@ namespace winrt::Microsoft::Terminal::Control::implementation
             const auto fullHeight{ ScrollBarCanvas().ActualHeight() };
             const auto totalBufferRows{ update.newMaximum + update.newViewportSize };
 
+            auto drawPip = [&](const auto row, const auto rightAlign, const auto& brush) {

```

Comment:
_LOL_, `auto`. you could `drawPip("e", 1.3f, nullptr)` and it would dutifully attempt to produce a lambda that supports that.

Reply:
It's only weird for the boolean, since it's the thing that is the most convertible.